recruit/records.go

```diff
@@ -0,0 +1,232 @@
+	Data []struct {
+		Salary         interface{} `json:"Salary"`
+		CurrencySymbol string      `json:"$currency_symbol"`
+		AccountManager struct {
+			Name string `json:"name"`
+			ID   string `json:"id"`
+		} `json:"Account_Manager"`
+		NoOfCandidatesHired int       `json:"No_of_Candidates_Hired"`
+		TargetDate          string    `json:"Target_Date"`
+		LastActivityTime    time.Time `json:"Last_Activity_Time"`
+		Industry            string    `json:"Industry"`
+		ModifiedBy          struct {
+			Name string `json:"name"`
+			ID   string `json:"id"`
+		} `json:"Modified_By"`
+		ProcessFlow     bool        `json:"$process_flow"`
+		ExpectedRevenue interface{} `json:"Expected_Revenue"`
+		IsHotJobOpening bool        `json:"Is_Hot_Job_Opening"`
+		AccountName     struct {
+			Name string `json:"name"`
+			ID   string `json:"id"`
+		} `json:"Account_Name"`
+		ZipCode       interface{} `json:"Zip_Code"`
+		ID            string      `json:"id"`
+		Skillset      string      `json:"Skillset"`
+		Approved      bool        `json:"$approved"`
+		Publish       bool        `json:"Publish"`
+		DateOpened    string      `json:"Date_Opened"`
+		PotentialName string      `json:"Potential_Name"`
+		Approval      struct {
+			Delegate bool `json:"delegate"`
+			Approve  bool `json:"approve"`
+			Reject   bool `json:"reject"`
+			Resubmit bool `json:"resubmit"`
+		} `json:"$approval"`
+		ModifiedTime             time.Time     `json:"Modified_Time"`
+		ActualRevenue            interface{}   `json:"Actual_Revenue"`
+		CreatedTime              time.Time     `json:"Created_Time"`
+		Followed                 bool          `json:"$followed"`
+		NoOfCandidatesAssociated int           `json:"No_of_Candidates_Associated"`
+		Editable                 bool          `json:"$editable"`
+		IsLocked                 bool          `json:"Is_Locked"`
+		City                     interface{}   `json:"City"`
+		JobOpeningStatus         string        `json:"Job_Opening_Status"`
+		RevenuePerPosition       int           `json:"Revenue_per_Position"`
+		ContactName              interface{}   `json:"Contact_Name"`
+		AssociatedTags           []interface{} `json:"Associated_Tags"`
+		AssignedRecruiter        []interface{} `json:"Assigned_Recruiter"`
+		MissedRevenue            interface{}   `json:"Missed_Revenue"`
+		JobOpeningID             string        `json:"Job_Opening_ID"`
+		JobDescription           string        `json:"Job_Description"`
+		WorkExperience           interface{}   `json:"Work_Experience"`
+		JobType                  string        `json:"Job_Type"`
+		State                    interface{}   `json:"State"`
+		NumberOfPositions        string        `json:"Number_of_Positions"`
+		Country                  interface{}   `json:"Country"`
+		CreatedBy                struct {
+			Name string `json:"name"`
+			ID   string `json:"id"`
+		} `json:"Created_By"`
+		IsAttachmentPresent bool `json:"Is_Attachment_Present"`
+	} `json:"data"`
+	Info PageInfo `json:"info,omitempty"`
+}
```

Comment:
I may be misreading the API docs, but I believe this is another `interface{}` response type. The endpoint seems to allow "associating" any record with other records. The Job Openings & Candidates seems to be just an example.

Looking at the "Associate Candidate" endpoint, I could be completely wrong. What do you think?